packages/stores/src/ui-config/slippage-config.ts

```diff
@@ -64,15 +64,16 @@ export class ObservableSlippageConfig {
 
   @action
   setManualSlippage(str: string) {
+    if (isNaN(Number(str))) return;
```

Comment:
Thoughts on using `isNumeric` instead?

Reply:
Where can that be imported from?

Reply:
Given this is the stores package, I don't have access to utils in web package.

Reply:
This function is a good candidate for `@osmosis-labs/utils`. This suggestion is a nit, so feel free to merge or move it to the utils folder